### ISSUE TITLE
Is there a way to get and set the content offset from the navigation bar?

### ISSUE THREAD

**Issue by hukjordanjanaq** [tzef/BmoViewPager#21]
I would like yo get and set the content offset from the navigation bar to get where is the scroll and sync with another navigation bar

**Comment by tzef** [comment]
Fair point, I added an observable property `contentOffset`, Can you take a try with version 5.2.0?
you can refer the example in `DemoViewController1.swift`
```
        self.contentOffsetObserver = ...
```

**Comment by hukjordanjanaq** [comment]
Thank you so much. It works! 
So for setting the content offset, it should be with the collectionView?

` self.contentOffsetObserver = defaultViewPagerNavigationBar.observe(\.contentOffset, options: [.new], changeHandler: { (collectionView, value) in
            guard let offset = value.newValue else { return }
            print("DEBUG: offset of navigation bar \(offset)")
        })`

Thanks for your work. It's a great library.
